src/profiler.rs

```diff
@@ -1216,6 +1222,10 @@ impl Profiler<'_> {
     }
 
     fn should_profile(&self, pid: i32) -> bool {
+        if self.exclude_self && (pid == std::process::id().try_into().unwrap()) {
```

Comment:
nit: no need for the parenthesis, and as we know that the type fits, we can use `as` until we have our own type

```suggestion
        if self.exclude_self && pid == std::process::id() as i32 {
```

Reply:
Fixed